### PR TITLE
update release leads for 1.3

### PR DESCRIPTION
**Pull request by psschwei**
# Changes

Updates release leads for 1.3

/cc @carlisia @xtreme-sameer-vohra @dprotaso

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -66,6 +66,7 @@ aliases:
   - pierDipi
   - vaikas
   knative-admin:
+  - carlisia
```
Comment (vaikas) [inline OWNERS_ALIASES:69]: Was this intentional? Didn't see it in the PR description.

**Comment by vaikas** [comment]
/lgtm
/approve
/hold

Just added the hold if the addition to admin was not intentional. If it was, please remove the hold.

File: OWNERS_ALIASES (hunk 1 above)
Comment (psschwei) [reply OWNERS_ALIASES:69]: That was done by the [update-codegen](https://github.com/knative/community/blob/main/hack/update-codegen.sh#L27-L28) script; I assume it's needed to do things like cut branches for the repos.

Comment (dprotaso) [reply OWNERS_ALIASES:69]: looks like this is done since the release team is a sub team of the admin group

**Pull request merged**